bundles/AdminBundle/Controller/AdminController.php

```diff
@@ -31,6 +31,15 @@
 
 abstract class AdminController extends Controller implements AdminControllerInterface
 {
+    /**
+     * @var TokenStorageUserResolver
+     */
+    protected $tokenResolver;
```

Comment:
please also remove form `getSubscribedServices()` as not needed anymore to load from service container.